src/components/About.js

```diff
@@ -57,7 +57,7 @@ class AboutDialog extends React.Component {
                 <ul className='list-unstyled'>
                   <li id={`${idPrefix}-version`}>Version <strong id={`${idPrefix}-version-value`}>{Product.version}-{Product.release}</strong></li>
                   <li id={`${idPrefix}-apiversion`}>{fixedStrings.BRAND_NAME} API Version <strong id={`${idPrefix}-apiversion-value`}>{apiVersion}</strong></li>
-                  <li id={`${idPrefix}-issues`}>Please report issues on <strong><a href='https://github.com/oVirt/ovirt-web-ui/issues' target='_blank' id={`${idPrefix}-issues-link`}>GitHub Issue Tracker</a></strong></li>
+                  <li id={`${idPrefix}-issues`}>Please report issues on <strong><a href={fixedStrings.BRAND_NAME === 'oVirt' ? fixedStrings.GITHUB_ISSUES_TRACKER_URL : fixedStrings.REDHAT_CUSTOMER_PORTAL_URL} target='_blank' id={`${idPrefix}-issues-link`}>{fixedStrings.BRAND_NAME === 'oVirt' ? 'GitHub Issue Tracker' : 'Redhat Customer Portal'}</a></strong></li>
```

Comment:
Like this, for example:
```JavaScript
<li id={`${idPrefix}-issues`}>Please report issues on <strong><a href={fixedStrings.ISSUES_TRACKER_URL} target='_blank' id={`${idPrefix}-issues-link`}>{fixedStrings.ISSUES_TRACKER_NAME}</a></strong></li>
```

Reply:
ohhh and then it will automatically change because RHV fixedStrings will override oVirt fixedStrings?

Reply:
RHV fixedStrings replace oVirt

Reply:
oh I forgot to add fixedstrings to the commit...